services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -583,13 +584,26 @@ class MediaColumn(Column):
     transform_column: type[Column]
 
     @classmethod
-    def transform(cls, example: dict[str, Any]) -> Any:
+    def transform(cls, example: dict[str, Any], is_struct: bool = True) -> Any:
         """
         Function to use to transform the original values to further pass these transformed values to statistics
         computation. Used inside ._compute_statistics() method.
         """
         raise NotImplementedError
 
+    @classmethod
+    def is_struct_type(cls, filename: Path, column_name: str) -> bool:
+        column_type = pq.read_schema(filename).field(column_name).type
+        if pa.types.is_struct(column_type):
+            return True
+        elif pa.types.is_binary(column_type) or pa.types.is_null(column_type):
+            return False
+        else:
+            raise ValueError(
```

Comment:
idk if i should do a type check and raise here actually

Reply:
if you use isinstance on the examples you might not need this function